data/saas/dataset/checkr_dataset.yml

```diff
@@ -0,0 +1,123 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: checkr
+    description: A sample dataset representing the Checkr connector for Fides
+    collections:
+      - name: user
+        fields:
+          - name: data
+            fidesops_meta:
+              data_type: "object[]"
+            fields:
+              - name: id
```

Comment:
We can get rid of this `data` field between `user` and the rest of the fields since we're unwrapping the response